test/publish.sh

```diff
@@ -27,55 +27,66 @@ else
   echo "No credential helpers found, push to docker may not function properly"
 fi
 
-DOCKER_HUB=${DOCKER_HUB:-gcr.io/istio-testing}
-GCS_BUCKET=${GCS_BUCKET:-istio-build/test}
-HELM_BUCKET=${HELM_BUCKET:-istio-build/test/charts}
-VERSION="1.14.0-releasebuilder.$(git rev-parse --short HEAD)"
+GCS_BUCKET=${GCS_BUCKET:-istio-prerelease/prerelease}
+HELM_BUCKET=${HELM_BUCKET:-istio-prerelease/charts}
 COSIGN_KEY=${COSIGN_KEY:-}
+GITHUB_ORG=${GITHUB_ORG:-istio}
+DOCKER_HUB=${DOCKER_HUB:-docker.io/istio}
```

Comment:
Is there any particular reason this is switched from `gcr.io` to `docker.io`?

Reply:
the build.sh script is a presubmit job and is set to "DOCKER_HUB=${DOCKER_HUB:-docker.io/istio}" so my reasoning is that the test/publish.sh job is a postsubmit job and it seemed to me that this test should validate what was created from the build job.

Reply:
this is the rest of publish, not the real publish. We don't want to go to the real dockerhub

Reply:
reverted DOCKER_HUB